packages/yoroi-extension/features/mock-dApp-webpage/index.js

```diff
@@ -185,6 +185,24 @@ export class MockDAppWebpage {
           callback({ success: false, errMsg: error.message });
         });
     });
+
+    await this.driver.executeScript(accResp => {
+      if (accResp.success) window.walletConnected = true;
+    }, accessResponse);
```

Comment:
Hey @Nebyt , sorry I didn't have time to review it earlier. The code looks correct but seems it can be simpler: just adding the line `window.walletConnected = true;` after line 181 would be OK.

Reply:
Will be fixed in the next PR.